tests/serverpod_test_client/lib/src/protocol/entities_with_list_relations/organization.dart

```diff
@@ -0,0 +1,86 @@
+/* AUTOMATICALLY GENERATED CODE DO NOT MODIFY */
+/*   To generate run: "serverpod generate"    */
+
+// ignore_for_file: library_private_types_in_public_api
+// ignore_for_file: public_member_api_docs
+// ignore_for_file: implementation_imports
+
+// ignore_for_file: no_leading_underscores_for_library_prefixes
+import 'package:serverpod_client/serverpod_client.dart' as _i1;
+import '../protocol.dart' as _i2;
+
+abstract class Organization extends _i1.SerializableEntity {
+  Organization._({
+    this.id,
+    required this.name,
+    this.people,
+  });
+
+  factory Organization({
+    int? id,
+    required String name,
+    List<_i2.Person>? people,
+  }) = _OrganizationImpl;
+
+  factory Organization.fromJson(
+    Map<String, dynamic> jsonSerialization,
+    _i1.SerializationManager serializationManager,
+  ) {
+    return Organization(
+      id: serializationManager.deserialize<int?>(jsonSerialization['id']),
+      name: serializationManager.deserialize<String>(jsonSerialization['name']),
+      people: serializationManager
+          .deserialize<List<_i2.Person>?>(jsonSerialization['people']),
+    );
+  }
+
+  /// The database id, set if the object has been inserted into the
+  /// database or if it has been fetched from the database. Otherwise,
+  /// the id will be null.
+  int? id;
+
+  String name;
+
+  List<_i2.Person>? people;
+
+  Organization copyWith({
+    int? id,
+    String? name,
+    List<_i2.Person>? people,
+  });
+  @override
+  Map<String, dynamic> toJson() {
+    return {
+      'id': id,
+      'name': name,
+      'people': people,
+    };
+  }
+}
+
+class _Undefined {}
```

Comment:
Question: Would it be possible for us to define this in some library so that it doesn't have to be generated in all client files? Possibly true on the server as well.